likecoin/public/iscn-badge.php

```diff
@@ -31,17 +31,34 @@
  * @param WP_Post| $post Post object.
  */
 function likecoin_add_iscn_badge( $post ) {
-	$post_id               = $post->ID;
-	$iscn_info             = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
-	$iscn_hash             = isset( $iscn_info['iscn_hash'] ) ? $iscn_info['iscn_hash'] : null;
-	$option                = get_option( LC_PUBLISH_OPTION_NAME );
-	$should_add_iscn_badge = checked( $option[ LC_OPTION_SITE_ADD_ISCN_BADGE ], 1, false );
+	$post_id   = $post->ID;
+	$iscn_info = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
+	$iscn_hash = isset( $iscn_info['iscn_hash'] ) ? $iscn_info['iscn_hash'] : null;
+	$option    = get_option( LC_PUBLISH_OPTION_NAME );
 
-	if ( strlen( $iscn_hash ) > 0 && $should_add_iscn_badge ) {
+	$is_dark_badge   = 0; // default is showing white badge.
+	$show_iscn_badge = true;
+
+	if ( isset( $option[ LC_OPTION_ISCN_BADGE_STYLE_OPTION ] ) ) {
+		$iscn_badge_style_option = $option[ LC_OPTION_ISCN_BADGE_STYLE_OPTION ];
+		switch ( $iscn_badge_style_option ) {
+			case 'dark':
+				$is_dark_badge = 1;
+				break;
+			case 'white':
+				break;
+			case 'none':
+				$show_iscn_badge = false;
+				break;
```

Comment:
Default case is `true` seems weird, I suggest default should be false, and setting `show_iscn_badge` to `true` to when case is `dark` and `white`

Reply:
ok!